modules/tinymce/src/themes/silver/main/ts/ui/toolbar/CommonToolbar.ts

```diff
@@ -27,10 +28,10 @@ export interface ToolbarSpec {
   onEscape: (comp: AlloyComponent) => Option<boolean>;
   initGroups: ToolbarGroup[];
   attributes?: Record<string, string>;
+  backstage: UiFactoryBackstage;
```

Comment:
You don't need the entire backstage here, so we should only pass through what's needed. In this case that's just the providers.

- [ ] Only pass through what's needed, not the entire backstage

Reply:
The entire backstage is being passed through to `partToolbar` and `partMultipleToolbar` in `/modules/tinymce/src/themes/silver/main/ts/Render.ts`. So, I am assuming it is needed?

Reply:
As discussed on slack, it's not actually required by any of that and it seems someone passed through the backstage to the `OuterContainer` specs previously. I've provided @wonderbugtiny a patch that shows we only need to pass the providers, not the entire backstage.